scipy/sparse/linalg/eigen/_svds.py

```diff
@@ -0,0 +1,269 @@
+        Matrix to decompose.
+    k : int, default: 6
+        Number of singular values and singular vectors to compute.
+        Must satisfy ``1 <= k < min(M, N)``.
+    ncv : int, optional
+        When ``solver='arpack'``, this is the number of Lanczos vectors
+        generated. See :ref:`'arpack' <sparse.linalg.svds-arpack>` for details.
+        When ``solver='lobpcg'``, this parameter is ignored.
+    tol : float, optional
+        Tolerance for singular values. Zero (default) means machine precision.
+    which : {'LM', 'SM'}
+        Which `k` singular values to find: either the largest magnitude ('LM')
+        or smallest magnitude ('SM') singular values.
+    v0 : ndarray, optional
+        The starting vector for iteration; see method-specific
+        documentation (:ref:`'arpack' <sparse.linalg.svds-arpack>` or
+        :ref:`'lobpcg' <sparse.linalg.svds-lobpcg>`) for details.
+    maxiter : int, optional
+        Maximum number of iterations; see method-specific
+        documentation (:ref:`'arpack' <sparse.linalg.svds-arpack>` or
+        :ref:`'lobpcg' <sparse.linalg.svds-lobpcg>`) for details.
+    return_singular_vectors : bool or str, optional
+        Singular values are always computed and returned; this parameter
+        controls the computation and return of singular vectors.
+
+        - ``True``: return singular vectors.
+        - ``False``: do not return singular vectors.
+        - ``"u"``: only return the left singular values, without computing the
+          right singular vectors (if ``N > M``).
+        - ``"vh"``: only return the right singular values, without computing
+          the left singular vectors (if ``N <= M``).
+
+    solver : str, optional
+            The solver used.
+            :ref:`'arpack' <sparse.linalg.svds-arpack>` and
+            :ref:`'lobpcg' <sparse.linalg.svds-lobpcg>` are supported.
+            Default: `'arpack'`.
+    options : dict, optional
+        A dictionary of solver-specific options. No solver-specific options
+        are currently supported; this parameter is reserved for future use.
+
+    Returns
+    -------
+    u : ndarray, shape=(M, k)
+        Unitary matrix having left singular vectors as columns.
+        If `return_singular_vectors` is ``"vh"``, this variable is not
+        computed, and ``None`` is returned instead.
+    s : ndarray, shape=(k,)
+        The singular values.
+    vh : ndarray, shape=(k, N)
+        Unitary matrix having right singular vectors as rows.
+        If `return_singular_vectors` is ``"u"``, this variable is not computed,
+        and ``None`` is returned instead.
+
+    Notes
+    -----
+    This is a naive implementation using ARPACK or LOBPCG as an eigensolver
+    on ``A.conj().T @ A`` or ``A @ A.conj().T``, depending on which one is more
+    efficient.
+
+    Examples
+    --------
+    Construct a matrix ``A`` from singular values and vectors.
+    >>> from scipy.stats import ortho_group
```

Comment:
```suggestion

    >>> from scipy.stats import ortho_group
```
I think this is the problem (everywhere like it)